src/VuePictureSwipe.vue

```diff
@@ -10,7 +10,9 @@
           itemtype="http://schema.org/ImageObject"
           v-for="(item, index) in items" :src="item.src"
           v-bind:key="index">
-        <a :href="item.src" itemprop="contentUrl" :data-size="'' + item.w + 'x' + item.h" :title="item.title">
+        <a 
+          v-show="nbThumbnailsDisplayed === -1 || index < nbThumbnailsDisplayed"
```

Comment:
This v-show should be in the figure props. Putting it in the <a> causes a bug in the display when there are too much thumbnails, because there are all on the sme line, which causes the last visible thumbnail to go on the next line.